modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/metadata/SourceMetadataTestCase.java

```diff
@@ -64,4 +66,11 @@ public void getSourceDynamicOutputMetadata() throws Exception {
     assertExpectedOutput(componentMetadata.getModel(), personType, typeLoader.load(StringAttributes.class));
     assertThat(componentMetadata.getMetadataAttributes().getKey().get(), is(PERSON_METADATA_KEY));
   }
+
+  @Test
+  public void sourcesMustNotStartWhenResolvingMetadata() {
```

Comment:
after asserting it is stopped, also test for starting it

Reply:
why?  I don't see the point